homeassistant/components/airvisual/__init__.py

```diff
@@ -192,7 +192,12 @@ async def async_setup_entry(hass, config_entry):
         cloud_api = CloudAPI(config_entry.data[CONF_API_KEY], session=websession)
 
         async def async_update_data():
-            """Get new data from the API."""
+            # Update interval periodically
```

Comment:
Small nit, but technically, we're synchronizing the update interval every time we call the API.

Reply:
I propose the following : 
`# ReAssess the interval between 2 requests to Cloud Server`
Do you agreee ?
NB: the comments was bigger before, but I fall on too many constraints from "pylint and flake8"
![image](https://user-images.githubusercontent.com/59493499/112730049-97ec1b00-8f2f-11eb-913d-dc73f6b064ff.png)